packages/core/src/components/button/_button.scss

```diff
@@ -65,6 +66,10 @@ Styleguide components.button.css
   @each $intent, $colors in $button-intents {
     &.pt-intent-#{$intent} {
       @include pt-button-intent($colors...);
+
+      .pt-button-spinner .pt-spinner-head {
```

Comment:
why is this inside the `@each` loop but the dark block on line 178 below isn't?

_much_ preferable to define this static rule once instead of redefining for each intent.